src/program_files/merchants/amazon_gift_card_reload.py

```diff
@@ -131,29 +131,37 @@ def web_automation(driver, merchant, amount):
             break
         except WebDriverException:
             pass
-
-    driver.find_element_by_xpath("//button[starts-with(text(),'Reload') and contains(text(),'" + utils.cents_to_str(amount) + "')]").click()
-    time.sleep(1 + random.random() * 2)
-
-    time.sleep(10)  # give page a chance to load
-    if 'thank-you' not in driver.current_url:
-        WebDriverWait(driver, 30).until(expected_conditions.element_to_be_clickable((By.XPATH, "//input[@placeholder='ending in " + merchant.card[-4:] + "']")))
-        elem = driver.find_element_by_xpath("//input[@placeholder='ending in " + merchant.card[-4:] + "']")
-        time.sleep(1 + random.random() * 2)
-        elem.send_keys(merchant.card)
-        time.sleep(1 + random.random() * 2)
-        elem.send_keys(Keys.TAB)
-        time.sleep(1 + random.random() * 2)
-        elem.send_keys(Keys.ENTER)
-        WebDriverWait(driver, 30).until(expected_conditions.element_to_be_clickable((By.XPATH, "//button[contains(text(),'Reload $" + utils.cents_to_str(amount) + "')]")))
-        time.sleep(1 + random.random() * 2)
+    
+    result = Result.success
+    if merchant.dry_run == False:
         driver.find_element_by_xpath("//button[starts-with(text(),'Reload') and contains(text(),'" + utils.cents_to_str(amount) + "')]").click()
+        time.sleep(1 + random.random() * 2)
```

Comment:
no idea why github is so bad at rendering this diff... it's the worst conflict to fix every time too